<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Nurse</class>
 <widget class="QWidget" name="Nurse">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1763</width>
    <height>987</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>731</width>
     <height>831</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>10</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Patlent Info</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>1420</x>
     <y>10</y>
     <width>295</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <widget class="QPushButton" name="Patient_New_pushButton">
      <property name="text">
       <string>New</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="Patient_Edit_pushButton">
      <property name="text">
       <string>Edit</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="Patient_Delete_pushButton">
      <property name="text">
       <string>Delete</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>1209</x>
     <y>50</y>
     <width>511</width>
     <height>341</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="Patient_Mobile_label">
        <property name="text">
         <string>Mobile</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="5">
       <widget class="QLineEdit" name="Patient_Mobile_lineEdit"/>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="Patient_phone_label">
        <property name="text">
         <string>Phone</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" colspan="4">
       <widget class="QLineEdit" name="Patient_Phone_lineEdit"/>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="Patient_Email_label">
        <property name="text">
         <string>E-mail</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="Patient_Email_lineEdit"/>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>@</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QLineEdit" name="lineEdit_7"/>
      </item>
      <item row="2" column="5">
       <widget class="QComboBox" name="Patient_comboBox">
        <item>
         <property name="text">
          <string>naver.com</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>gmail.com</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout_3" columnminimumwidth="0,0">
      <item row="0" column="0">
       <widget class="QLabel" name="Patient_Address_label">
        <property name="text">
         <string>Address</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="Patient_Address_lineEdit"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="Patient_DetailAddress_lineEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="Patient_DetailAddress_label">
        <property name="text">
         <string>Detail Address</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>754</x>
     <y>50</y>
     <width>399</width>
     <height>341</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="1" colspan="3">
     <widget class="QDateEdit" name="Patient_FirstVist_dateEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="Patient_Gender_label">
      <property name="text">
       <string>Gender</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1" colspan="3">
     <widget class="QDateEdit" name="Patient_LastVist_dateEdit"/>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="Patient_Gender_None_pushButton">
      <property name="text">
       <string>None</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="Patient_FirstVist_label">
      <property name="text">
       <string>First Vist</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="Patient_LastVist_label">
      <property name="text">
       <string>Last Vist</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="Patient_ChartNumber_label">
      <property name="text">
       <string>Chart Number</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="Patient_Gender_Female_pushButton">
      <property name="text">
       <string>Female</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="3">
     <widget class="QLineEdit" name="Patient_Birthdat_lineEdit"/>
    </item>
    <item row="0" column="1" colspan="3">
     <widget class="QLineEdit" name="Patient_Name_lineEdit"/>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="Patient_Gender_Male_pushButton">
      <property name="text">
       <string>Male</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="Patient_Birthday_label">
      <property name="text">
       <string>Birthday</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="Patient_Name_label">
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="3">
     <widget class="QLineEdit" name="Patient_ChartNumber_lineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>1163</x>
     <y>60</y>
     <width>20</width>
     <height>331</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QWidget" name="Patient_lsit_widget" native="true">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>410</y>
     <width>961</width>
     <height>41</height>
    </rect>
   </property>
   <widget class="QLabel" name="Patlent_list_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>수술목록</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Patient_List_attachment_pushButton">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>0</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>첨부파일</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="Patient_Data_widget" native="true">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>660</y>
     <width>961</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QLabel" name="Patient_Data_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Data</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Patient_Data_pushButton">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>0</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>첨부파일</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>490</y>
     <width>791</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QOpenGLWidget" name="Patient_Project_List_openGLWidget"/>
    </item>
    <item>
     <widget class="QOpenGLWidget" name="Patient_Project_List_openGLWidget2"/>
    </item>
    <item>
     <widget class="QOpenGLWidget" name="Patient_Project_List_openGLWidget3"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>700</y>
     <width>801</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QOpenGLWidget" name="Patient_Data_upper_openGLWidget"/>
    </item>
    <item>
     <widget class="QOpenGLWidget" name="Patient_Data_lower_openGLWidget"/>
    </item>
    <item>
     <widget class="QOpenGLWidget" name="Patient_Data_occlusion_openGLWidget"/>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>Patient_Data_widget</zorder>
  <zorder>Patient_lsit_widget</zorder>
  <zorder>listView</zorder>
  <zorder>label_6</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>line</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
